### PR TITLE
Colab standardize restarts

### PR DESCRIPTION
**Pull request by zeroasterisk**
copy/paste 3 json blocks into colabs for "search" and "gemini" for runtime restart standardization.  

I spot-checked 2 of the colabs and they are working correctly.

Want to land this PR before editing more of the colabs.

Related to issue #280

### CODE REVIEW

**Comment by lavinigam-gcp** [comment]
Thank you, Alan, for raising the PR. I have marked respective owners of the notebook as the reviewer.

File: embeddings/intro-textemb-vectorsearch.ipynb [hunk 1]
```diff
@@ -345,27 +345,57 @@
...
+     "data": {
+      "text/plain": [
+       "{'status': 'ok', 'restart': True}"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```
Comment (holtskinner) [inline embeddings/intro-textemb-vectorsearch.ipynb:377]: Not sure this cell should be added to the repo version?

**Review by holtskinner — APPROVED** [review]
Thanks for standardizing these.

Just a minor comment about a specific executed cell to confirm it should be added.

**Comment by holtskinner** [comment]
The lint error is a false alarm. I'm adding a commit for a .gitleaksignore to handle this.

**Pull request merged**